client/src/redux/selectors/proverbSelectors.js

```diff
@@ -0,0 +1,7 @@
+export const selectProverb =  (state) => ({
+    proverbs: state.proverb.proverbs,
+    userProverbs: state.proverb.userProverbs,
+    proverb: state.proverb.proverb,
+    loading: state.proverb.loading,
+    error: state.proverb.error,
+  })
```

Comment:
```suggestion
export const selectProverb =  (state) => state.proverb
```

Here you can also simplify the selector.